futaba/client.py

```diff
@@ -99,6 +102,14 @@ def run_with_token(self):
         Sets up the bot's state, loads cogs then prints a 'ready' message.
         """
 
+        # Get error channel
+        if self.config.error_channel_id:
+            for channel in self.get_all_channels():
```

Comment:
Would [get_channel](https://discordpy.readthedocs.io/en/rewrite/api.html#discord.Client.get_channel) not look better here?

Reply:
Oh, good point.